tests/e2e/utils/lambda_layer/powertools_layer.py

```diff
@@ -16,9 +16,16 @@ class LocalLambdaPowertoolsLayer(BaseLocalLambdaLayer):
 
     def __init__(self, output_dir: Path = CDK_OUT_PATH):
         super().__init__(output_dir)
-        self.package = f"{SOURCE_CODE_ROOT_PATH}[pydantic]"
+        self.package = f"{SOURCE_CODE_ROOT_PATH}[all]"
         self.build_args = "--platform manylinux1_x86_64 --only-binary=:all: --upgrade"
         self.build_command = f"python -m pip install {self.package} {self.build_args} --target {self.target_dir}"
+        self.cleanup_command = (
+            f"rm -rf {self.target_dir}/boto* {self.target_dir}/s3transfer* && "
+            f"rm -rf {self.target_dir}/*dateutil* {self.target_dir}/urllib3* {self.target_dir}/six* && "
```

Comment:
```suggestion
            f"rm -rf {self.target_dir}/boto* {self.target_dir}/s3transfer* && "
            f"rm -rf {self.target_dir}/*dateutil* {self.target_dir}/urllib3* {self.target_dir}/six* && "
            f"rm -rf {self.target_dir}/jmespath* && "
```

We forgot `jmespath` as it comes with [`botocore`](https://github.com/boto/botocore/blob/develop/setup.py#L26). This means we don't need that as an optional either (I'll comment in `pyproject.toml` directly)

Reply:
Note to self: apply this to the cdk construct too